compiler/rustc_resolve/src/check_unused.rs

```diff
@@ -514,8 +516,59 @@ impl Resolver<'_, '_> {
             }
         }
 
+        let mut redundant_imports = FxIndexSet::default();
```

Comment:
```suggestion
        let mut redundant_imports = UnordSet::default();
```
This set doesn't need ordering.